contrib/standalone-installer/make.py

```diff
@@ -85,8 +85,31 @@ def download_cab_file (directory, uri):
     subprocess.run (cmd, cwd=directory, check=True)
 
 def download_flatpak (directory):
-    cmd = ['wget', 'https://people.freedesktop.org/~hughsient/temp/fwupd.flatpak', '-O', 'fwupd.flatpak']
-    if 'DEBUG' in os.environ:
+    # check if installed yet
+    dep = 'org.freedesktop.fwupd'
+    cmd = ['flatpak', 'info', dep]
+    verbose = 'DEBUG' in os.environ
+    with open(os.devnull, 'w') as devnull:
+        if not verbose:
+            output = devnull
+        else:
+            output = None
+        if verbose:
+            print(cmd)
+        ret = subprocess.run (cmd, stdout=output, stderr=output)
+        #not installed
```

Comment:
This looks a bit hacky to me; isn't there something we can just check for existence for? e.g. `~/.local/share/flatpak/app/org.freedesktop.fwupd/x86_64/stable` ?

Reply:
The problem is it could be installed in system or in user and you'd have to check both

Reply:
I don't think it would build the bundle if it's installed systemwide as you're pointing `build-bundle` at the user repo ; and you can certainly have the same thing installed per-user and per-system.

Reply:
Ah I see, I didn't realize you could install both user and system with the same flatpak.  If I keep this this way (pending other comment) I'll switch around and just probe for the directory.

Reply:
I've adjusted this.